src/graphics/program-lib/programs/standard.js

```diff
@@ -339,7 +339,7 @@ const standard = {
                     func.append("getSpecularityFactor();");
                 }
                 if (options.useSpecularColor) {
-                    code.append(this._addMap("specular", "specularPS", options, litShader.chunks));
+                    code.append(this._addMap("specular", "specularPS", options, litShader.chunks, options.specularEncoding));
```

Comment:
I think we want this:

```suggestion
                    code.append(this._addMap("specular", "specularPS", options, litShader.chunks, options.useMetalness ? 'srgb' : 'linear'));
```